selfdrive/sensord/pigeond.py

```diff
@@ -136,6 +143,67 @@ def reset_device(self) -> bool:
         return True
     return False
 
+  def get_gnss_config(self):
+    # UBX-CFG-GNSS pull request
+    hdr = b'\xb5\x62\x06\x3E'
+    self.send(hdr + b'\x00\x00\x44\xD2')
+    while True:
+      recv = self.receive()
+      if hdr in recv:
+        return hdr + recv.split(hdr)[1]
+
+  def set_gnss_config(self):
+    # UBX-CFG-GNSS
+    hdr  = b'\xB5\x62\x06\x3E'
+    c    = b'\x00\x1c\x1c'
+    msg  = struct.pack("I", GPS_MAX_TRACK<<16 | GPS_RES_TRACK<<8) # GPS GNSS CFG
+    msg += b'\x01\x00\x01\x01'
+    msg += b'\x01\x00\x00\x00' # GPS SBAS (disable)
+    msg += b'\x00\x00\x01\x01'
+    msg += b'\x02\x00\x00\x00' # GALILEO (disable)
+    msg += b'\x00\x20\x00\x01'
+    msg += b'\x03\x00\x00\x00' # BEIDOU (disable)
+    msg += b'\x00\x00\x01\x01'
+    msg += b'\x04\x00\x00\x00' # IMES (disable)
+    msg += b'\x00\x00\x01\x01'
+    msg += struct.pack("I", QZSS_MAX_TRACK<<16 | QZSS_RES_TRACK<<8 | 5) # QZSS (keep enabled with GPS)
```

Comment:
```suggestion
    msg += struct.pack("I", (QZSS_MAX_TRACK << 16) | (QZSS_RES_TRACK << 8) | 5) # QZSS (keep enabled with GPS)
```

Just my opinion but I would use parenthesis instead of spaces to denote the order of ops